src/WebJobs.Script.WebHost/Security/Authentication/Jwt/ScriptJwtBearerExtensions.cs

```diff
@@ -53,10 +53,16 @@ public static AuthenticationBuilder AddScriptJwtBearer(this AuthenticationBuilde
                     },
                     OnTokenValidated = c =>
                     {
-                        c.Principal.AddIdentity(new ClaimsIdentity(new Claim[]
+                        var claims = new List<Claim>
                         {
                             new Claim(SecurityConstants.AuthLevelClaimType, AuthorizationLevel.Admin.ToString())
-                        }));
+                        };
+                        if (!string.Equals(c.SecurityToken.Issuer, ScriptConstants.AppServiceCoreUri, StringComparison.OrdinalIgnoreCase))
+                        {
+                            claims.Add(new Claim(SecurityConstants.InvokeClaimType, bool.TrueString));
```

Comment:
Open to suggestions on whether this value should be "True" or perhaps "1" or something